node/tessel.js

```diff
@@ -376,6 +379,48 @@ Pin.prototype.readPulse = function(type, timeout, callback) {
     throw new Error("Pin.readPulse is not yet implemented");
 }
 
+var ANALOG_RESOLUTION = 4096;
+Pin.prototype.resolution = ANALOG_RESOLUTION;
+
+Pin.prototype.analogRead = function (cb) {
+    if (!this.analogSupported) {
+        console.warn("pin.analogRead is not supoprted on this pin. Analog read is supported on port A pins 4 and 7 and on all pins on port B");
+        return this;
+    }
+
+    if (typeof cb != "function") {
+        console.warn("analogPin.read is async, pass in a callback to get the value");
+    }
+
+    this._port.cork();
```

Comment:
cork/uncork are unnecessary since there's only a single write to the socket between them.